### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if appI…

### PR DESCRIPTION
**Pull request by OVikhrov**
…D value type
@aarotar please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua]: @OVikhrov, please update Requirement summary with at least one related functional requirement

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+-- b) OnSeekMediaClockTimer notification allowed in preloaded file for default app
+-- c) App successfylly registered, consented and activated
+--
+-- 2. Performed steps:
+-- a) HMI sends OnSeekMediaClockTimer notification to SDL with wrong appID value type (string).
+--
+-- Expected result:
+-- a) SDL does not resend OnSeekMediaClockTimer notification to mobile app.
+---------------------------------------------------------------------------------------------
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_With_Wrong_appID_Value_Type()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime = {hours = 30, minutes = 30, seconds = 30}, appID = tostring(self.applications["Test Application"])})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua (hunk 3 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_appID_Wrong_Value_Type.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**